microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/publickeylocation/PublicKeyFileTestCase.java

```diff
@@ -0,0 +1,4 @@
+package org.jboss.eap.qe.microprofile.jwt.security.publickeylocation;
+
+public class PublicKeyFileTestCase {
```

Comment:
I guess this one is empty because it will be needed for next iteration, right? Just to be sure.

Reply:
Yep, sorry, it shouldn't be included in this commit, will remove. Thank you!